cargo-shuttle/src/args.rs

```diff
@@ -399,6 +402,7 @@ mod tests {
             from: None,
             subfolder: None,
             create_env: false,
+            force_name: false,
```

Comment:
It might be worth refactoring this to somehow create this struct so that we don't have to update a bunch of tests each time a new field is added.